test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTestsFixture.cs

```diff
@@ -1,46 +1,21 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-using System.Runtime.InteropServices;
-using DotNet.Testcontainers.Containers;
 using Testcontainers.MsSql;
-using Testcontainers.SqlEdge;
 using Xunit;
 
 namespace OpenTelemetry.Instrumentation.SqlClient.Tests;
 
 public sealed class SqlClientIntegrationTestsFixture : IAsyncLifetime
 {
-    // The Microsoft SQL Server Docker image is not compatible with ARM devices, such as Macs with Apple Silicon.
-    // See https://github.com/microsoft/mssql-docker/issues/881.
-    public IContainer DatabaseContainer { get; } = Architecture.Arm64.Equals(RuntimeInformation.ProcessArchitecture) ? CreateSqlEdge() : CreateMsSql();
+    public MsSqlContainer DatabaseContainer { get; } = CreateMsSql();
 
-    public Task InitializeAsync()
-    {
-        return this.DatabaseContainer.StartAsync();
-    }
+    public Task InitializeAsync() => this.DatabaseContainer.StartAsync();
 
-    public Task DisposeAsync()
-    {
-        return this.DatabaseContainer.DisposeAsync().AsTask();
-    }
-
-    private static SqlEdgeContainer CreateSqlEdge()
-    {
-        // Note: The Testcontainers.SqlEdge package has been deprecated. Seems
-        // it will not work with newer GitHub-hosted runners. Need to find an
-        // alternative solution. See:
-        // https://github.com/testcontainers/testcontainers-dotnet/pull/1265
-        return new SqlEdgeBuilder().Build();
-    }
+    public Task DisposeAsync() => this.DatabaseContainer.DisposeAsync().AsTask();
 
     private static MsSqlContainer CreateMsSql()
-    {
-        // Note: This "WithImage" line can most likely be removed when there is
-        // a new version (>3.10.0) of Testcontainers.MsSql released. See:
-        // https://github.com/testcontainers/testcontainers-dotnet/pull/1265
-        return new MsSqlBuilder()
-            .WithImage("mcr.microsoft.com/mssql/server:2022-CU20-ubuntu-22.04")
-            .Build();
-    }
+        => new MsSqlBuilder()
+               .WithImage("mcr.microsoft.com/mssql/server:2022-CU20-GDR1-ubuntu-22.04")
```

Comment:
What's about suggestion that this can be removed with newer testconainer package?

Reply:
Potential improvement, tested in autoinstrumentation:
Reference [version ](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/9296b0aedbd516dadfded1cb917c3ec245ec5b16/test/IntegrationTests/KafkaCollection.cs#L23) from [local dockerfile](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/9296b0aedbd516dadfded1cb917c3ec245ec5b16/test/IntegrationTests/docker/kafka.Dockerfile). Dockerfile can be updated by dependabot/renovate.

Separate PR if you consider it as useful.

Reply:
Cool, I'll do that in a later PR.